app/pages/notifications/comment.cjsx

```diff
@@ -11,17 +14,36 @@ module.exports = React.createClass
     project: React.PropTypes.object
     user: React.PropTypes.object.isRequired
 
+  getDefaultProps: ->
+    startedDiscussion: false
+
   getInitialState: ->
     comment: null
 
   render: ->
+    notification = @props.notification
+    comment = @props.data.comment
+    commentUser = @props.data.commentUser
+
     if @props.data.comment
-      <div className="talk-comment">
-        <Comment
-          data={@props.data.comment}
-          user={@props.user}
-          project={@props.project}
-          params={@props.params} />
+      <div className="conversation-message talk-module">
+        {if notification.delivered is false and !@props.startedDiscussion
+          <i title="Unread" className="fa fa-star fa-lg" />}
+
+        <Link to={notification.url} {...@props} className="message-link">
```

Comment:
Quick note on this tag. In production, this is a straight link to the board and comment, but in staging, a strange prefix is added `demo.zooniverse.org`, I believe.